### PR TITLE
Fixes for alerts

### PR DESCRIPTION
**Pull request by louismaximepiton**
Proposal to fix v4 -> v5. 

- [x] **Docs > Components > Alerts** [Sizes](https://boosted.orange.com/docs/5.1/components/alerts/#sizes): remove margin when stacked? (no design specs).
  - divide by 2 the margin, could maybe be reduced more ?
- [x] **Docs > Components > Alerts** Missing specific [Boosted message about the icons from v4](https://boosted.orange.com/docs/4.6/components/alerts/#examples).
  - Added the same as v4 one.
- [x] **Docs > Components > Alerts** Missing specific [Boosted message about the div](https://boosted.orange.com/docs/4.6/components/alerts/#additional-content) in "Additional content"
  - Added the same as v4 one.

### CODE REVIEW

File: scss/_alert.scss [hunk 1]
```diff
@@ -79,7 +79,7 @@
 //
 
 //// Additional content
-.alert * p {
+.alert p {
```
Comment (MewenLeHo) [inline scss/_alert.scss:82]: As discussed, this may cause strange behaviours xith the following rule:
``
.alert-icon+* {
    margin: 0.125rem 0;
}
``

**Review by MewenLeHo — CHANGES_REQUESTED** [review]

File: site/content/docs/5.1/components/alerts.md [hunk 2]
```diff
@@ -95,6 +101,8 @@ Alerts come with a smaller variant: `.alert-sm`.
 
 Alerts can also contain additional HTML elements like headings, links and paragraphs.
 
+As of Boosted, it’s recommended to wrap your additional content in a `<div>` to ensure proper alignment — and, for headings, to always use the .alert-heading class.
```
Comment (MewenLeHo) [inline site/content/docs/5.1/components/alerts.md:104]: Not sure about Unicode?

**Review by MewenLeHo — COMMENTED** [review]
Just one comment

**Review by MewenLeHo — APPROVED** [review]
👌

**Comment by julien-deramond** [comment]
@louismaximepiton normal that it's been closed?

File: site/content/docs/5.1/components/alerts.md [hunk 3]
```diff
@@ -10,6 +10,12 @@ toc: true
 
 Alerts are available for any length of text, as well as an optional close button. For proper styling, use one of the four **required** contextual classes (e.g., `.alert-success`). For inline dismissal, use the [alerts JavaScript plugin](#dismissing).
 
+Boosted also adds a dedicated icon for each contextual class using `.alert-icon`, matching [functional colors in palette]({{<docsref "utilities/colors#accessibility">}}):
```
Comment (julien-deramond) [inline site/content/docs/5.1/components/alerts.md:13]: The link shouldn't be more something like "/docs/5.x/customize/color/"?

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**